### PR TITLE
Add support for capitalised proxy environment variables

### LINKED ISSUE #22: Include capitalised versions of environment variables
Some programs check for the `HTTP_PROXY`, `HTTPS_PROXY` and environment variables like this instead of the usual lowercase ones. One example I could give right now is the _Ruby_ package manager `gem` [(see this stackoverflow answer)](https://stackoverflow.com/questions/4418/how-do-i-update-ruby-gems-from-behind-a-proxy-isa-ntlm).

### PR DESCRIPTION
**Pull request by SkullTech**
Fixes #22 

### Modified files
- `environment.sh`
- `bash.sh`

### CODE REVIEW

**Comment by SkullTech** [comment]
Any updates @himanshub16  ?

**Comment by SkullTech** [comment]
Done. Used case-insensitive `grep` in the `list_proxy` functions.

File: apt.sh [hunk 1]
```diff
@@ -54,7 +54,6 @@ list_proxy() {
 	if [ "$lines" -gt 0 ]; then
 		cat /etc/apt/apt.conf | grep proxy -i | sed -e "s/Acquire//g" -e "s/\:\:/\ /g" -e "s/\;//g"
 	else
-		echo -e "\e[36m None \e[0m"
```
Comment (himanshub16) [inline apt.sh]: Why is this missing?

Comment (SkullTech) [inline apt.sh]: Must have been a mistake. I did it in a hurry and maybe that's why. I'll fix it.

**Comment by himanshub16** [comment]
Thanks for your contributions. :+1: 
I'll test it once and revert you back to you.

**Pull request merged**